source/headers.ts

```diff
@@ -29,6 +31,40 @@ const getResetSeconds = (
 	return resetSeconds
 }
 
+/**
+ * Returns a humanised version of the duration of the window, to be used to name
+ * the rate limiter in the policy header.
+ *
+ * @param windowMs {number} - The window length.
+ */
+const getDurationInWords = (windowMs: number): string => {
+	const seconds = windowMs / 1000
+	const minutes = windowMs / (1000 * 60)
+	const hours = windowMs / (1000 * 60 * 60)
+	const days = windowMs / (1000 * 60 * 60 * 24)
+
+	if (seconds < 60) return `${seconds}sec`
+	if (minutes < 60) return `${minutes}min`
+	if (hours < 24) return `${hours}hrs`
+
+	return `${days}day`
+}
```

Comment:
Nice!

I like that this will have a decimal for values that don't divide evenly, but omit it otherwise. (E.g. `1000` -> `'1sec'`, and `1500` -> `'1.5sec'`)